tests/Unit/Routes/WebRoutesTest.php

```diff
@@ -1,20 +1,153 @@
 <?php
 
-namespace Tests\Unit\Routes;
-
 use Tests\TestCase;
 use Illuminate\Foundation\Testing\DatabaseMigrations;
 
 class WebRoutesTest extends TestCase
 {
+    use DatabaseMigrations;
+
+    private $user = null;
+    private $centre = null;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->centre = factory(App\Centre::class)->create();
+
+        // Create a User
+        $this->user =  factory(App\User::class)->create([
+            "name"  => "test user",
+            "email" => "testuser@example.com",
+            "password" => bcrypt('test_user_pass'),
+            "centre_id" => $this->centre->id,
+        ]);
```

Comment:
And a random role? Or do you want one of each?

Reply:
The distinction for the route tests are "logged in, or not" so the actual role is irrelevant for this set.